INN/umbrella-citylimits
# remove text-transform: uppercase on links on special projects page text widgets

## Changes

This pull request makes the following changes:

- Removes `text-transform: uppercase;` styles.

![Screen Shot 2020-02-04 at 19 23 46](https://user-images.githubusercontent.com/1754187/73799452-0d24eb80-4784-11ea-8748-c307991af7ab.png)


## Why



For #146 


## Testing/Questions

Features that this PR affects:

- links in a specific widget setup on special project pages


Questions that need to be answered before merging:


Steps to test this PR:

1. check it out
2. create a text widget matching the contents of the text widget added to the bottom of https://citylimits.org/series/futuremap/
